packages/connect-wallet-modal/src/components/Ledger/helpers.ts

```diff
@@ -0,0 +1,57 @@
+import { LS_KEY_DERIVATION_PATH } from '@reef-knot/ledger-connector';
+import { LedgerContextValue } from './LedgerContext';
+import { DERIVATION_PATHS } from './constants';
+
+export async function getTransport() {
+  return await (await import('@ledgerhq/hw-transport-webhid')).default.create();
+}
```

Comment:
you actually do not need to write `return await`, it will be handled anyway
```suggestion
export const getTransport = () => (await import('@ledgerhq/hw-transport-webhid')).default.create();
```